stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/MessagesScreen.kt

```diff
@@ -61,7 +62,7 @@ public fun MessagesScreen(
     showHeader: Boolean = true,
     enforceUniqueReactions: Boolean = true,
     onBackPressed: () -> Unit = {},
-    onHeaderActionClick: () -> Unit = {},
+    onHeaderActionClick: (Channel) -> Unit = {},
```

Comment:
Small tip for later: you can name these params and it will help a bit with autocompletion on the caller side:

```suggestion
    onHeaderActionClick: (channel: Channel) -> Unit = {},
```

Reply:
@zsmb13 Yeah that's a good idea - do you think we should do this for all of our lambdas?

Reply:
Yeah, I think it's good practice. Wish there was a lint warning to enable for this 😃 It's a super minor thing though, really just helps the IDE tooling a tiny bit on the call site. Doesn't have any practical effect otherwise.